hil/cli/port.py

```diff
@@ -11,9 +13,30 @@ def port():
 @port.command(name='show')
 @click.argument('switch')
 @click.argument('port')
-def port_show(switch, port):
+@click.option('--jsonout', is_flag=True)
+def port_show(switch, port, jsonout):
     """Show what's connected to <port>"""
-    print client.port.show(switch, port)
+
+    raw_output = client.port.show(switch, port)
+
+    if jsonout:
+        json_output = json.dumps(raw_output)
+        print(json_output)
+        return
+
+    port_table = PrettyTable()
+    port_table.title = 'PORT TABLE'
+    port_table.field_names = ['ATTRIBUTE', 'INFORMATION']
+
+    if 'node' in raw_output:
+        port_table.add_row(['Node', raw_output['node']])
+    if 'nic' in raw_output:
+        port_table.add_row(['NIC', raw_output['nic']])
+    if 'networks' in raw_output:
+        port_table.add_row(['Networks', raw_output['networks'].keys()[0]])
+        port_table.add_row(['', raw_output['networks'].values()[0]])
```

Comment:
This way you are only displaying one network. There can be multiple networks. Also, you could format the output for this part like how you did `node show`.
```
|        PORT TABLE       |
+-----------+-------------+
| ATTRIBUTE | INFORMATION |
+-----------+-------------+
|    Node   |   neu-15-3  |
|    NIC    |     nic1    |
|  Networks | vlan/native |
|           |   bmi-pro   |
+-----------+-------------+

```
This is how it should be (excuse the formatting):
```
|        PORT TABLE       |
+-----------+-------------+
| ATTRIBUTE | INFORMATION |
+-----------+-------------+
|    Node   |   neu-15-3  |
|    NIC    |     nic1    |
|  Networks | bmi-pro(vlan/native) |
|           |   internet(vlan/204)      |
+-----------+-----------------------------+

```